src/views/Main/Container.js

```diff
@@ -16,7 +16,7 @@ export class Container extends React.Component {
 
   constructor() {
     super();
-    Loom.configure({ baseUrl: 'localhost' });
+    Loom.configure({ baseUrl: 'api.loom.digital' });
```

Comment:
not sure if it's a good idea of hardcoding this. let's not rush, and talk about it first.